src/utils.js

```diff
@@ -26,7 +26,12 @@ export function convertClientNameToId(name, clients) {
 }
 
 export function convertClientNamesToIds(names, clients) {
-  return names.map(name => convertClientNameToId(name, clients));
```

Comment:
Is `convertClientNameToId` referenced anywhere else, can it be removed? Or could that be modified instead?

Reply:
it's actually used in https://github.com/auth0-extensions/auth0-source-control-extension-tools/blob/bdefbbdcc39e8b18a98fce90c8fbef39e5f6b9d1/src/auth0/handlers/connections.js#L41 

I'm not sure if this too may be affected by duplicate client names